.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: ruby
+os:
+- linux
+- osx
+rvm:
+- 2.5
+before_install:
+- gem install metanorma-cli
+script:
```

Comment:
Why do we need to hard code this metanorma commands in the Travis file?

Reply:
@abunashir what is your concern?

Reply:
What's the purpose of having the `metanorma new test/iso-rice --type iso --doctype iso-rice` as a script? since the executable is available in the build, wouldn't it make it easier to run it in the fly using `system` or any other command executor,  that way we also have more flexibility to pass any arguments as necessary?

Reply:
> What's the purpose of having the metanorma new test/iso-rice --type iso --doctype iso-rice as a script?

To test future changes on this repo, it hasn't any ruby code, just docs/assets

> since the executable is available in the build

Which build you mean?

> wouldn't it make it easier to run it in the fly using system or any other command executor, that way we also have more flexibility to pass any arguments as necessary?

With bash I have the same level of flexibility, isn't it?

Reply:
@CAMOBAP795: Sorry, my bad. I was pre-assuming lots of things here and most likely I was thinking of how I would have approached it to write the test, which I shouldn't not :)

It's still CI setup, so please ignore this for now and go ahead with your tests and we can revisit this discussion later if necessary, thanks!